notebooks/useful_notebooks/01_find_and_release.ipynb

```diff
@@ -380,8 +430,16 @@
     "                    if cnt:\n",
     "                        patch_data['extra_files'] = extras\n",
     "\n",
-    "            if action:\n",
-    "                ff_utils.patch_metadata(patch_data, obj_id=raw_data['uuid'],key=my_auth)\n",
+    "        # Additional changes\n",
```

Comment:
I think that not every schema has tags and also at least a couple (ontology_term, lab and award themselves do not have attribution or viewable_by for that matter.  I think this would only be an issue when you are trying to patch released items - maybe we should not allow that although we have wanted to add tags to released items for sure.  An alternative would be to add a check to see if the property exists in the schema before adding it to the patch. Maybe a function that checks if a_type schema has the property (by check /profiles/a_type.json?